src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export const getVisibleMovies = (
+  query: string,
+  listOfMovies: Movie[],
+) => {
+  return listOfMovies.filter((movie: Movie) => {
+    const { description, title } = movie;
+    const filteredByDescription = description
+      .toLowerCase()
+      .includes(query.toLowerCase().trim());
+    const filteredByTitle = title
+      .toLowerCase()
+      .includes(query.toLowerCase().trim());
```

Comment:
Maybe you have an idea, how minimaze this similiar code by creating separate variable or function)

Reply:
agree